src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -306,14 +306,14 @@ object Helpers extends SparkSessionWrapper {
     tablesPar.foreach(tbl => {
       try {
         val zorderColumns = if (tbl.zOrderBy.nonEmpty) s"ZORDER BY (${tbl.zOrderBy.mkString(", ")})" else ""
-        val sql = s"""optimize ${tbl.tableFullName} $zorderColumns"""
-        println(s"optimizing: ${tbl.tableFullName} --> $sql")
+        val sql = s"""optimize delta.`${tbl.tableLocation}` $zorderColumns"""
```

Comment:
maybe optimize each organization_id separately if necessary? Otherwise we'll rewrite a lot of data in the multi-workspace setup

Reply:
optimize and zorder are done within partitions, right? Since every target / table is partitioned by organization_id all optimization workloads should be confined to the partition directory[ies], right?

Reply:
Yes, but if people will run it from multiple workspaces, it will run for all partitions

Reply:
right, I will try to make that clear in the docs. If I limit it to a single org_id here then users will be REQUIRED to run this on every workspace which is not what I want them to do.